keyboards/willow/willow64/keymaps/default/keymap.c

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2021 Hanachi
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include QMK_KEYBOARD_H
+
+#define _QWERTY 0
+#define _FN1 1
+#define _FN2 2
+#define _FN3 3
```

Comment:
We highly recommend using enums for the layer naming as it is easier to manage long term. Especially as people will copy the default. 

```suggestion
enum custom_layers {
    _QWERTY,
    _FN1,
    _FN2,
    _FN3,
};
```